### ISSUE TITLE
ONNX integration between PyTorch and CNTK

### ISSUE THREAD

**Issue by lext** [MIPT-Oulu/3D-Histo-Grading#5]
- [ ] Bilinear upsampling as a transposed convolution
- [ ] BatchNorm Fusion
- [ ] Compare the inference results (CNTK vs PyTorch)
- [ ] Tiling strategy for large samples
- [ ] Test VNet and UNet

**Comment by jfrondel** [comment]
Upsampling has been added in  97eccdc

**Comment by jfrondel** [comment]
Model weights are saved after bn fusion with pytorch. CNTK model loads fused weights.

**Comment by jfrondel** [comment]
Inference works almost identically between cntk and pytorch. Difference in the dice scores are in the 3rd and 4th decimal place. bn fusion was omitted from the model due to issues. Working inference was added in  96cf1da.

**Comment by jfrondel** [comment]
bn fusion was fixed in  2dbb59a.
